app/artifact-cas/internal/conf/conf.pb.go

```diff
@@ -15,7 +15,7 @@
 
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.30.0
+// 	protoc-gen-go v1.31.0
```

Comment:
Because of the version change of protoc, I have other files that have this kind of modification.
Would you like for me to add all the files that were re-generated with the new version? Or only keep the ones that are relevant to the configuration only?

Reply:
Thanks for asking. I'd keep it like this, meaning just adding to the patch what's relevant, otherwise the review will get too much noise. 

In another patch, we can bump the generated files if we want, wdyt?

Reply:
Sounds good 👍